mesheryctl/pkg/utils/helpers.go

```diff
@@ -547,7 +547,10 @@ func ClearLine() {
 		clearCmd = exec.Command("cmd", "/c", "cls") // for Windows
 	}
 	clearCmd.Stdout = os.Stdout
-	clearCmd.Run()
+	err := clearCmd.Run()
+	if err != nil {
```

Comment:
@gyohuangxin @MUzairS15 I wonder if we should be using a MeshKit Error here. https://docs.meshery.io/project/contributing/contributing-error

Reply:
What's your perspective, @MUzairS15?

Reply:
Yes, the usage of meshkit error is appropriate

Reply:
@leecalcote @MUzairS15 It's a good idea, can we create another issue to track this? So that we can fix the security issues as quickly as possible. What do you think?

Reply:
Ok, sure.

Reply:
Here we are - https://github.com/meshery/meshery/issues/10343